haystack/components/embedders/sentence_transformers_text_embedder.py

```diff
@@ -46,6 +46,7 @@ def __init__(  # noqa: PLR0913 # pylint: disable=too-many-positional-arguments
         progress_bar: bool = True,
         normalize_embeddings: bool = False,
         trust_remote_code: bool = False,
+        revision: Optional[str] = None,
```

Comment:
Same here let's move this one to the bottom of the init since this init method consists of positional arguments